### PR TITLE
Feature/bookmarking

### PR DESCRIPTION
**Pull request by elnzv**
Closing #232

### CODE REVIEW

**Comment by elnzv** [comment]
@apinf/developers please review this.

File: client/widgets/favourite.js [hunk 1]
```diff
@@ -7,7 +7,28 @@ Template.favourite.events({
...
+    var bookmarkIndex = apiIds.indexOf(backendId);
+
+    return (bookmarkIndex >= 0) ? true : false;
+  }
+});
+
+Template.favourite.created = function () {
+  // subscribe to user bookmarks
+  this.subscribe('myApiBookmarks');
+  this.subscribe('myBookmarkedApis');
```
Comment (brylie) [inline client/widgets/favourite.js]: This subscription may be unnecessary. Try commenting out and testing again.

Comment (brylie) [inline client/widgets/favourite.js]: For readability and understanding, rename the publications to be more clear. E.g. 'userApiBookmarks' and 'userBookmarkedApiBackends'.

File: server/methods/apiBookmarks.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 Meteor.methods({
-  "bookmarkApi": function (backendId, currentUserId) {
+  "toggleBookmarkApi": function (backendId, currentUserId) {
 
     var apiBackendIds = [];
     apiBackendIds.push(backendId);
```
Comment (brylie) [inline server/methods/apiBookmarks.js]: Add comments describing what is going on each step of the way.

File: server/methods/apiBookmarks.js [hunk 3]
```diff
@@ -11,28 +11,30 @@ Meteor.methods({
     if (existingUserBookmarks) {
       var apiIds = existingUserBookmarks.apiIds;
```
Comment (brylie) [inline server/methods/apiBookmarks.js]: Add comment here.

**Comment by frenchbread** [comment]
`< Code >` and `// comments` look good! Merging..

**Pull request merged**